geoengine/types.py

```diff
@@ -326,6 +327,17 @@ def __repr__(self) -> str:
 
         return r
 
+    # TODO: add to super class as well
```

Comment:
You can do that :-)

Reply:
6e9e1769a6797bfcfb6e32ac5b8afcd7ee4b7be4